sport/app/rugby/views/fragments/matchStats.scala.html

```diff
@@ -0,0 +1,76 @@
+
+    @defining((matchStat.get.teams.head, matchStat.get.teams.last)) { case (firstTeam, secondTeam) =>
+
+     <div data-component="football-stats-embed" class="@RenderClasses(Map(
+                "c-match-stats" -> true
+            ))">
+
+    <h3 class="component__heading content__meta-heading">Match stats</h3>
+
+
+    <h4 class="match-stats__caption">Possession</h4>
+    <div class="football-possession">
+        <table class="match-stats bar-fight js-chart"
+                   data-chart-type="doughnut"
+                   data-chart-unit="%"
+                   data-chart-class="chart--football-possession"
+                   data-chart-width="200"
+                   data-chart-show-values="true">
+                <thead hidden>
+                    <tr>
+                        <th>@firstTeam.name</th>
+                        <th>@secondTeam.name</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                    @defining((firstTeam.possession * 100.toInt).toInt, (secondTeam.possession * 100).toInt) { case (firstPossession, secondPossession) =>
+                        <td
+                            class="bar-fight__bar bar-fight__bar--home"
+                            data-chart-value="@secondPossession"
+                            data-chart-color="#rgb(0, 171, 77)"
+                            style="background-color: rgb(0, 171, 77); width: @NudgePercent(secondPossession, firstPossession)%;">@secondPossession</td>
+                        <td
+                            class="bar-fight__bar bar-fight__bar--away"
+                            data-chart-value="@firstPossession"
+                            data-chart-color="rgb(0, 83, 166)"
+                            style="background-color: rgb(0, 83, 166); width: @NudgePercent(firstPossession, secondPossession)%;">@firstPossession</td>
+                    }  
+                    </tr>
+                </tbody>
+        </table>
+    </div>
+
+    @barfight("Meters carried", firstTeam.carries_metres, secondTeam.carries_metres)
+    @barfight("Tackles made", firstTeam.tackles, secondTeam.tackles)
+    @barfight("Tackles missed", firstTeam.missed_tackles, secondTeam.missed_tackles)
+    @barfight("Turnovers won", firstTeam.turnover_won, secondTeam.turnover_won)
+    @barfight("Penalties conceded", firstTeam.penalties_conceded, secondTeam.penalties_conceded)
+
+    </div>
+    }
+}
+
+@barfight(propertyTitle: String, firstTeamProperty: Int, secondTeamProperty: Int) = {
+    <dl class="u-cf match-stats bar-fight">
+            <dt class="match-stats__caption">@propertyTitle</dt>
+            @defining(PercentMaker(firstTeamProperty, secondTeamProperty)){ case (firstPercent, secondPercent) =>
+                <dd 
+                    class="bar-fight__bar bar-fight__bar--home"
+                    style="background-color: rgb(0, 83, 166); width: @NudgePercent(firstPercent, secondPercent)%;">
```

Comment:
Please note that the colours used are fixed and use the [IRB ones](http://www.rugby15.co.za/wp-content/uploads/2012/12/Rugby-World-Cup-2015.jpg)